webapi/webapi-iap-xwalk-tests/index.html

```diff
@@ -0,0 +1,5 @@
+<!doctype html>
```

Comment:
This file is unnecessary, it is automatically generated while test suite packing.